src/chart/line/LineView.ts

```diff
@@ -725,10 +727,10 @@ class LineView extends ChartView {
                 }
             }
 
-            polyline = this._newPolyline(points);
+            polyline = this._newPolyline(points, closed);
```

Comment:
It's a better idea to define `const closed = isCoordSysPolar && seriesModel.get('closed');` here. It will make the code more readable and postpone the execution until necessary.